tests/test_store.py

```diff
@@ -0,0 +1,36 @@
+from asu.config import settings
+from asu.fastapi.staticfiles import FileResponse
+
+store_path = settings.public_path / "store"
```

Comment:
This is ugly.

But conftest.py overwrites settings.public_path, unfortunately the main.py/app has already been constructed, with the previous old public_path.
Thus the conftest.py app()-setting of public_path to a temp-path doesn't work here.